File: Algorithms/Warmup/Solve me second/solve_me_second.c
#include <stdio.h>

int sum(int a, int b);

int main(){
	int T;
	int i;
	int num1, num2;
	scanf("%d", &T);

	for(i = 0; i < T; i++){
		scanf("%d %d", &num1, &num2);
		printf("%d", sum(num1, num2));
	}

	return 0;

}

int sum(int a, int b){
	return (a + b);
}

File: Algorithms/Warmup/Solve me second/SolveMeSecond.java
import java.util.Scanner;
public class SolveMeSecond{
	static int sum(int a, int b){
		return (a + b);
	}

	public static void main(String[] args){
		int T, i, num1, num2;
		Scanner in = new Scanner(System.in);

		T = in.nextInt();
		for(i = 0; i < T; i++){
			num1 = in.nextInt();
			num2 = in.nextInt();
			System.out.println(sum(num1, num2));
		}

	}
}